python/sparkdl/transformers/param.py

```diff
@@ -92,11 +97,58 @@ def getOutputCol(self):
         return self.getOrDefault(self.outputCol)
 
 
-# New in sparkdl
-
+"""
+TensorFlow Specific Parameters
```

Comment:
Remove this comment. As the name suggests, the converters are mean to be for SparkDL generally (though they are mostly for TensorFlow, there is one that's generically model name at the bottom, for example).